python/.env.example

```diff
@@ -8,3 +8,4 @@ AZURE_COGNITIVE_SEARCH_ADMIN_KEY=""
 PINECONE_API_KEY=""
 PINECONE_ENVIRONMENT=""
 POSTGRES_CONNECTION_STRING=""
+GOOGLE_API_KEY=""
```

Comment:
Can we rename this to GOOGLE_PALM_API_KEY?

Reply:
Yes